### DISCUSSION TITLE
Edge-tts

### DISCUSSION THREAD

**Discussion by RobertWalls1234** [rany2/edge-tts#114] [General]
I was wondering if you could use edge-tts in your main program file ie (ai project) or do you have to have it in a separate program file. Thank you for your response.

**Comment by rany2** [comment]
It depends, are you using Python for your AI Project? If you are you could use it directly by adding `edge-tts` to your required packages list and importing it as is done in https://github.com/rany2/edge-tts/tree/master/examples

If you are not using Python, you could try to call `edge-tts`'s CLI tool instead and just execute it as just about any other program.

**Reply by RobertWalls1234** [reply]
Thank you
